cookbooks/aws-parallelcluster-slurm/resources/munge_key_manager.rb

```diff
@@ -70,3 +70,58 @@
     end
   end
 end
+
+action :update_munge_key do
+  bash 'remove_current_munge_key' do
```

Comment:
Is there a reason why we want to remove the current munge key before trying to generate or fetch a new one?
I'm thinking about a case where the generation or fetch fails, we may not want to have an empty/non-existent `/etc/munge/munge.key` file.

Reply:
Have been safely removed.